README.md

```diff
@@ -441,6 +449,32 @@ Getting path to an executable based on a command name:
 console.log(await $.which("deno")); // outputs the path to deno executable
 ```
 
+Check if a command exists:
+
+```ts
+console.log(await $.commandExists("deno"));
+console.log($.commandExistsSync("deno"));
+```
+
+Check if a command does not exist:
+
+```ts
+console.log(await $.commandMissing("deno"));
+console.log($.commandMissingSync("deno"));
+```
+
+Check if an environment variable is set (and is not blank):
+
+```ts
+console.log($.envExists("HOME"));
```

Comment:
I'm not sure we want to add this one since people should probably be using `Deno.env` for this.

Reply:
Sounds good, this API has been removed from the PR :+1: